README.md

```diff
@@ -227,7 +224,9 @@ tox -e py39 -- tests/unit/test_cli.py::TestGenerateEnv::test_invalid_format
 tox -e py39 -- tests/unit/extras/test_envfile.py::test_cannot_determine_format
 ```
 
-In short, tox passes all arguments to the right of `--` directly to pytest.
+In short, tox passes all arguments to the right of `--` directly to pytest. For more information on
```

Comment:
I would rephrase this slightly to mention what `--` means. It is not something specific to tox only.

Reply:
Why? Explaining shell positional arguments and how it translates to tox `{posargs}` feels largely outside of the scope of user-facing docs on a tool that doesn't directly deal with that, IOW users don't need to understand shell in order to develop/test or even use this tool, so the original wording used in this case feels adequate for the desired outcome. Moreover, I only expanded the sentence providing references to other sources explaining the necessary internals should the user be interested in exploring them; note that I didn't try to explain how people should read a Dockerfile or how tox.ini works, so I'd preferably not like to take any action here.

Reply:
```suggestion
Double dash (--) signifies the end of command options, so any arguments after it are passed directly to pytest without further interpretation by tox.
```
...if it is not out of scope